### PR TITLE
wrong format of application.properties file

### PR DESCRIPTION
**Pull request by lsulak**
Will need to retrigger the ECR push once this is merged. But not sure if this really is causing the problems.

### CODE REVIEW

**Review by jakipatryk — APPROVED** [review]

File: server/Dockerfile [hunk 1]
```diff
@@ -53,12 +53,12 @@ ARG SCALA_VERSION=2.12
...
 
 # Copy Spring application properties
-COPY $CONFIG /opt/application.yaml
+COPY $CONFIG /opt/application.properties
 
 # deploy as root application in tomcat
 COPY ${AS_PREFIX}/target/jvm-${SCALA_VERSION}/*.war /usr/local/tomcat/webapps/ROOT.war
 
-ENV SPRING_CONFIG_LOCATION=/opt/application.yaml
+ENV SPRING_CONFIG_LOCATION=/opt/application.properties
```
Comment (benedeki) [inline server/Dockerfile:61]: Is this the correct location?

**Review by benedeki — COMMENTED** [review]
How are the actual values supplied into the `application.conf` file?

**Comment by lsulak** [comment]
> How are the actual values supplied into the `application.conf` file?

a brand new application.properties file is crafted and its path is supplied to the `SPRING_CONFIG_LOCATION`

File: server/Dockerfile (hunk 1 above)
Comment (lsulak) [reply server/Dockerfile:61]: @Zejnilovic used the same approach as in Login Service

**Review by benedeki — APPROVED** [review]

**Pull request merged**